src/components/SearchBar.tsx

```diff
@@ -1,21 +1,37 @@
-import { TextField as MaterialTextField } from '@material-ui/core';
+import { TextField as MaterialTextField, makeStyles, Theme } from '@material-ui/core';
+import { createStyles } from '@material-ui/core/styles';
 import React from 'react';
+import CloseIcon from '@material-ui/icons/Close';
 
 interface SearchProps {
   onSearchChange: (event: React.ChangeEvent<{ value: unknown }>) => void;
+  onSearchExit?: any;
   placeholder?: string;
 }
 
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'inline-flex',
+      width: '100%',
+    },
+  }));
+
 function SearchBar(props: SearchProps) {
+  const classes = useStyles(props);
+
   return (
-    <MaterialTextField
-      name="search"
-      onChange={props.onSearchChange}
-      variant="standard"
-      type="search"
-      fullWidth
-      placeholder={props.placeholder}
-    />
+    <div className={classes.root}>
+      <MaterialTextField
+        name="search"
+        onChange={props.onSearchChange}
+        variant="standard"
+        fullWidth
+        placeholder={props.placeholder}
+        inputRef={input => input && input.focus()}
```

Comment:
This line makes the SearchBar focus, regardless of where it's placed. I see how it's important in the BaseHeader use case, but in other places (this is clear when I ran it on this branch which still has a standalone `SearchBar`), it focuses the search bar upon opening the page, which probably isn't ideal behavior for using this component in other contexts. Maybe make something like `autoFocus` an optional prop that causes this behavior?